arm-network/2016-09-01/swagger/applicationGateway.json

```diff
@@ -1194,7 +1194,7 @@
       },
       "allOf": [
         {
-          "$ref": "./network.json#/definitions/Resource"
+          "$ref": "./network.json#/definitions/SubResource"
```

Comment:
This change looks suspicious to me. A subresource should not have a properties attribute. Can you please expand on why this change is made?